### DISCUSSION TITLE
Is it possible to link to code?

### DISCUSSION THREAD

**Discussion by napter** [foambubble/foam#1147] [Q&A]
I have .md files in my javascript project, can I reference a .js file in the code so that clicking it will open the file in vscode? When I type "[[" the autocomplete doesn't suggest my .js files.

**Comment by riccardoferretti** [comment]
Foam supports attachments.
The list of file extensions that are considered attachment is a setting, and by default it does not include `.js`, which is why you don't see it.
To be able to reference js files just add that extension to the list in `Files: Attachment Extensions`
